pkg/osquery/tables/dataflattentable/exec_and_parse.go

```diff
@@ -77,6 +82,9 @@ func NewExecAndParseTable(logger log.Logger, tableName string, p parser, execCmd
 }
 
 func (t *execTableV2) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	ctx, span := traces.New(ctx, trace.WithAttributes(attribute.String(traces.AttributeName("dataflattentable", "table_name"), t.tableName)))
```

Comment:
I _want_ to add spans to the generate, but I think we should be doing it at a different level.

Reply:
Where do you have in mind?

Reply:
Probably the osquery-go SDK, or even thrift.

Reply:
I imagine we'll come back to that idea a little later?